app/serializers/search/goods_nomenclature_serializer.rb

```diff
@@ -15,15 +15,15 @@ def serializable_hash(_opts = {})
           description:,
           description_indexed:,
           description_indexed_shingled: description_indexed,
-          formatted_description:,
+          formatted_description: formatted_description || '',
           search_references:,
           search_intercept_terms:,
           ancestors:,
           validity_start_date:,
           validity_end_date:,
           guides:,
           guide_ids:,
-          declarable: true,
+          declarable: declarable?,
```

Comment:
Not sure about this?
The tests 
created a non-declarable and was expecting a declarable false in the serialised, which makes sense.

But the code returned `true` hardcoded! 
My hunch is that this has not been run, since it was marked as "flaky".